Makefile

```diff
@@ -6,6 +6,9 @@ ROOTDIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 # CircleCI yarn cache directory may also need to be updated in sync with this
 YARN_CACHE_DIR=~/.yarncache
 
+ISOLATED_VM_VERSION_COMMAND="require('./node_modules/isolated-vm/package.json').version"
+ISOLATED_VM_VERSION=$(shell node -p -e $(ISOLATED_VM_VERSION_COMMAND))
```

Comment:
I don't think node is installed in many CircleCI images.  We typically rely on python commands for our build scripts.

Reply:
yeah I checked that we are using `cimg/node:14.13` for this repo's CI docker image. in fact most commands in this Makefile will fail if node is not available.